inc/update.php

```diff
@@ -514,217 +500,6 @@ function largo_update_custom_less_variables() {
 	}
 }
 
-/**
- * Checks for use of deprecated widgets and posts an alert
- */
-function largo_check_deprecated_widgets() {
```

Comment:
@benlk is it safe to complete remove this at this point or do we still need this in case, say, someone skips a version of largo and is updating from 0.4 to 0.5.5 or something edge-casey like that?